OutOfSchool/OutOfSchool.BusinessLogic/Services/CompetitiveEventService.cs

```diff
@@ -121,6 +121,55 @@ public async Task Delete(Guid id)
         }
     }
 
+    /// <inheritdoc/>
+    public async Task<SearchResult<CompetitiveEventViewCardDto>> GetByProviderId(Guid id, ExcludeIdFilter filter)
+    {
+        logger.LogDebug("Getting Competitive events by organization started. Looking ProviderId = {id}.", id);
+
+        filter ??= new ExcludeIdFilter();
+        ValidateExcludedIdFilter(filter);
+
+        var predicate = PredicateBuilder.True<CompetitiveEvent>();
+        predicate = predicate.And(x => x.OrganizerOfTheEventId == id);
+
+        if (filter.ExcludedId is not null)
+        {
+            predicate = predicate.And(x => x.Id != filter.ExcludedId);
+        }
+        var competitiveEventCardsCount = await competitiveEventRepository.Count(
+            whereExpression: predicate).ConfigureAwait(false);
+
+        var competitiveEvents = await competitiveEventRepository.Get(
+            skip: filter.From,
+            take: filter.Size,
+            includeProperties: null,
+            whereExpression: predicate)
+            .ToListAsync()
+            .ConfigureAwait(false);
+
+        var competitiveEventViewCards = mapper.Map<List<CompetitiveEventViewCardDto>>(competitiveEvents);
+
+        if (competitiveEventViewCards.Any())
```

Comment:
`logger.LogDebug("{Count} CompetitiveEvents were successfully received", competitiveEventViewCards.Count);` is more then enough, no need for if else just for logging. If `count = 0` anyone would understand that the list is empty :)